lib/bitreserve/entities/base_entity.rb

```diff
@@ -3,15 +3,25 @@ module Entities
     class BaseEntity
       include Virtus.model
 
-      def self.from_collection(entities)
-        entities.map do |entity|
-          new(entity)
-        end
+      def self.from_collection(entities, content_range)
```

Comment:
Actually, [here](https://developer.bitreserve.org/api/v0/#pagination) it states that:

> All collection endpoints support Range Pagination Headers

However, on the [release notes](https://github.com/bitreserve/api/releases/tag/2015-02-17) only a few endpoints are mentioned, not all.

@fixe Can you clarify this doubt for us?